README.adoc

```diff
@@ -215,6 +215,15 @@ list of available Rake tasks via:
 $ bundle exec rake -T
 ---------------------
 
+=== Code style
```

Comment:
Except code style is not about tests/testing. This are a set of rules to be followed which might or might not be enforced.